apps/test/unit/gamelab/GameLabTest.js

```diff
@@ -66,6 +67,44 @@ describe('GameLab', () => {
     describe('After being injected with a studioApp instance', () => {
       beforeEach(() => instance.injectStudioApp(studioApp));
 
+      it('Rerun mutes sounds', () => {
+        instance.gameLabP5.p5 = sinon.spy();
+        instance.gameLabP5.p5.allSprites = sinon.spy();
+        instance.gameLabP5.p5.allSprites.removeSprites = sinon.spy();
+        instance.gameLabP5.p5.redraw = sinon.spy();
+        instance.JSInterpreter = sinon.spy();
+        instance.JSInterpreter.deinitialize = sinon.spy();
+        instance.initInterpreter = sinon.spy();
+        instance.onP5Setup = sinon.spy();
+
+        let spy = sinon.stub(Sounds.getSingleton(), 'muteURLs');
```

Comment:
It looks like the sinon.stub-ing and restoring could be placed in a beforeEach/afterEach. You would probably need to nest these tests in their own describe to properly scope the beforeEach/afterEach.